app/client/packages/design-system/widgets/src/components/Select/src/SelectTrigger.tsx

```
@@ -32,10 +34,10 @@ export const SelectTrigger: React.FC<SelectTriggerProps> = (props) => {
             isPlaceholder ? placeholder : defaultChildren
           }
         </SelectValue>
+        <span data-input-suffix>
+          {Boolean(isLoading) ? <Spinner /> : <Icon name="chevron-down" />}
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)